### PR TITLE
Tweak cargo crate names

### PR DESCRIPTION
**Pull request by DamianX**
Assertions:
- Menu entries should NOT have "crate" at the end, it is superfluous.
- The actual crate that comes via shuttle SHOULD have the crate suffix, it is a crate, after all.
- Capitalization rules must be followed.

Given that, this PR does the following:
- Makes capitalization consistent
- ~~Makes all crate names \improper~~ Prefixes \improper where needed
- Removes the "crate" suffix from the menu entries
- Appends, where missing, the crate suffix to the containername
- Adjusts some names, fixes typos
- Leaves everything else the same (content, price, categories)

The one thing I'm not so sure about is the usage of \improper. Comments regarding that are more than welcome.

meme screenshot below:

:cl:
 * tweak: Tweaks cargo crate names and menu entries

### CODE REVIEW

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -42,21 +42,21 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
...
 /datum/supply_packs/labels
-	name = "Label Rolls"
+	name = "Label rolls"
 	contains = list(/obj/item/weapon/storage/box/labels,
 					/obj/item/weapon/storage/box/labels, //21 label rolls is enough to label Beepsky "SHITCURITRON" 375 times,
 					/obj/item/weapon/storage/box/labels) //so this might be a bit excessive.
 	cost = 10
 	containertype = /obj/structure/closet/crate
-	containername = "Label Rolls"
+	containername = "\improper label rolls crate"
```
Comment (Intigracy) [inline code/datums/supplypacks.dm]: If the word is already lowercase you don't need \improper

**Review by Intigracy — DISMISSED** [review]

**Comment by yclatious** [comment]
I kinda liked having it say crate in everything, but if you really don't like them, I guess ill manage. As a cargo dude, no complaints here.

**Comment by DamianX** [comment]
But it didn't. Some things said crate, some things said Crate, some things said something else.
Some entries still don't fit inside the window (which, by the way, even if you resize it, it doesn't expand the menu entry box width. Perhaps this too should be looked into too.), even after my changes, with the "crate" suffix it's even worse. 
That said, I don't feel THAT strongly about this whole thing, so I won't be offended if it doesn't get merged. Also, I fixed the \impropers in this last commit.

**Comment by yclatious** [comment]
I say everything but I meant most things, don't sweat it, I don't mind it much and I doubt any other cargo players would mind too much either way. Even if it doesn't get merged, it was a nice gesture with a good intention.

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**